app/controllers/sessions_controller.rb

```diff
@@ -7,9 +7,18 @@ class SessionsController < Devise::SessionsController
   # before_action :configure_sign_in_params, only: [:create]
 
   # GET /resource/sign_in
-  # def new
-  #   super
-  # end
+  def new
+    self.resource = resource_class.new(sign_in_params)
+    clean_up_passwords(resource)
+    yield resource if block_given?
+
+    if resource_class.omniauth_providers.empty?
+      render :new_with_no_providers
+    else
+      @local_account = params[:local] == 'true'
```

Comment:
We can just set local_account to params[:local]. Don't need to check if it's true, as we will only be passing in the local param if the link is for a local account

Reply:
I was weird and set the return button param to false. Fixed in fd8b1e3626721a48ecc73c1ae3717d6d6ed686ff, but I might have introduced an error message now. Does this happen for you too?
![Screenshot from 2023-10-12 14-08-23](https://github.com/phac-nml/irida-next/assets/325703/3f39e85c-e119-49f2-bd05-c7506dd48923)